easybuild/easyconfigs/s/SPAdes/SPAdes-3.10.1-intel-2016b.eb

```diff
@@ -0,0 +1,49 @@
+# This file is an EasyBuild reciPY as per https://github.com/hpcugent/easybuild
+# Author: Pablo Escobar Lopez
+# sciCORE - University of Basel
+# SIB Swiss Institute of Bioinformatics 
+# 3.9.0:
+# Modified by:
+# Adam Huffman
+# The Francis Crick Institute
+
+easyblock = "CMakeMake"
+
+name = 'SPAdes'
+version = '3.10.1'
+
+homepage = 'http://cab.spbu.ru/software/spades/'
+description = """Genome assembler for single-cell and isolates data sets"""
+
+toolchain = {'name': 'intel', 'version': '2016b'}
+
+source_urls = ['http://cab.spbu.ru/files/release%(version)s']
+sources = [SOURCE_TAR_GZ]
+
+patches = ['spades.patch']
+
+builddependencies = [
+    ('CMake', '3.7.2'),
+    ('Boost', '1.63.0', '-Python-2.7.12'),
```

Comment:
this may not be wise, since SPAdes includes an old version of Boost (1.51.0)... see also discussion in #4261